cloudformation/database/serverless-database.yaml

```diff
@@ -20,58 +20,58 @@ Metadata:
           - DatabaseClusterSubnetTwo
     ParameterLabels:
       Vpc:
-        default: "Which VPC should the database cluster be created in?"
+        default: "The VPC to use for the database cluster."
       ClusterAdmin:
-        default: "What should the database administrator user name be?"
+        default: "The database administrator user name."
       AdminPasswordSecretName:
-        default: "What should the name of the secret containing the administrator password be?"
+        default: "The name of the secret containing the administrator password."
       AdminPasswordSecretString:
-        default: "What should the administrator password be?"
+        default: "The administrator password."
       MinCapacity:
-        default: "What should the minimum scaling capacity of the database cluster be?"
+        default: "The minimum scaling capacity of the database cluster."
       MaxCapacity:
-        default: "What should the maximum scaling capacity of the database cluster be?"
+        default: "The maximum scaling capacity of the database cluster."
       DatabaseClusterSubnetOne:
-        default: "What is the first subnet the database cluster should be deployed in?"
+        default: "The first subnet to use for the database cluster."
       DatabaseClusterSubnetTwo:
-        default: "What is the second subnet the database cluster should be deployed in?"
+        default: "The second subnet to use for the database cluster."
 Parameters:
   AdminPasswordSecretName:
```

Comment:
Probably out of scope for this specific PR: feel free to address this comment in a different one.

I suggest to only provide `AdminPasswordSecretString` instead of both `AdminPasswordSecretString` and `AdminPasswordSecretName` because:
1. the template will be used mainly in the following scenarios: (1) tutorial (2) beta tests (3) integration tests. In all of them we don't want the extra step of manually creating a secret in SecretsManager.
1. the template is doing already a lot of useful things, let's keep it simpler where it is possible.

Reply:
Removed.